cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1587,4 +1587,52 @@
     </description>
   </property>
 
+  <!-- Dataset extensions -->
+  <property>
+    <name>dataset.extensions.dir</name>
+    <value>/opt/cdap/ext/lib</value>
+  </property>
+
+  <property>
+    <name>dataset.extensions.modules</name>
+    <value>name1:classname1,name2:classname2,...</value>
+  </property>
+
+  <!--
```

Comment:
If want to document all configurations that are used by the extensions system, better have a doc describing those. @awholegunch can provides guidance on how to do so.